amgut/templates/portal.html

```diff
@@ -159,7 +159,7 @@ <h2 style="width:500px;margin-right:0px;float:left;">{% raw tl['MAILING_HEADER_1
                     <form action="{% raw media_locale['SITEBASE'] %}/authed/sample_overview/" method="post" id="sample_overview2_{{ result['barcode'] }}">
                         <input type="hidden" name="barcode" id="barcode" value="{{ result['barcode'] }}"/>
                     </form>
-                    <a href="" onclick="document.forms['sample_overview2_{{ result['barcode'] }}'].submit(); return false;">{{ result['barcode'] }} ({{ escape(result['participant_name']) }})</a>
+                    <a href="" onclick="document.forms['sample_overview2_{{ result['barcode'] }}'].submit(); return false;">{{ result['barcode'] }} ({{ escape(result.get('participant_name', '') or '') }})</a>
```

Comment:
Is the reason why you have to do 'or' is because the value of that key could be associated with None?